lib/p2p/P2PRepository.ts

```diff
@@ -37,6 +37,14 @@ class P2PRepository {
   public addNodes = async (nodes: NodeFactory[]) => {
     return this.models.Node.bulkCreate(<NodeAttributes[]>nodes);
   }
+
+  public setBannedBy = async (nodePubKey: string, bannedBy: boolean) => {
```

Comment:
Maybe call this `updateBannedBy` to differentiate from the other `setBannedBy` method?